setup-issue-templates/workshop-release.md

```diff
@@ -1,3 +1,6 @@
 In [`AlexsLemonade/training-modules`](https://github.com/AlexsLemonade/training-modules), [create a new release](https://github.com/AlexsLemonade/training-modules/releases/new) that ties that material to this specific workshop. We use the convention `<year>-<month>` for tags and `<year> <month>` for release titles for Data Lab workshops.
 
-Don't forget to update the `_config.yaml` file in _this repository_ to point to the correct release.
+Be sure to update the `_config.yml` file _in this repository_ accordingly:
+
+- `release_tag` and `docker_tag` should point to the correct release
+- `docker_repo` should, in most cases, be filled in with `training_rnaseq`
```

Comment:
The docker repo has been updated by the automated build process to be more uniform (and less topic-specific)
In practice, we could probably just fill this in by default.
```suggestion
- `docker_repo` should, in most cases, be filled in with `training_rstudio`
```